livedata/src/main/java/io/getstream/chat/android/livedata/service/sync/BackgroundSyncConfigStore.kt

```diff
@@ -0,0 +1,59 @@
+package io.getstream.chat.android.livedata.service.sync
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.security.crypto.EncryptedSharedPreferences
+import androidx.security.crypto.MasterKey
+
+interface BackgroundSyncConfigStore {
+    fun put(config: BackgroundSyncConfig)
+    fun get(): BackgroundSyncConfig
+    fun clear()
+}
+
+class EncryptedBackgroundSyncConfigStore(context: Context) : BackgroundSyncConfigStore {
```

Comment:
This is an internal class that shouldn't be used by other devs, we should keep them `internal`